src/docs/releases/3.0.0.md

```diff
@@ -281,6 +281,25 @@ Additionally, when manually creating or updating content items in your project,
 
 A new **CollapseContainedItems** setting has been added to **BagPartSettings** and **FlowPartSettings**, allowing users to control whether contained items are collapsed or expanded by default. To ensure consistency, both **BagPart** and **FlowPart** now default to expanding contained items when the page loads.
 
+#### Content Manager Methods Updated
+Previously, the method
+`Task<IEnumerable<ContentItem>> GetAsync(IEnumerable<string> contentItemIds, bool latest = false)`
+in the `DefaultContentManager` class returned content items in the same order as the provided list of IDs.
```

Comment:
The change in https://github.com/OrchardCMS/OrchardCore/pull/16077 was introduced in 2.0.0 so please update that release notes.

Reply:
Agreed.